charts/evm-stack/Chart.lock

```diff
@@ -1,10 +1,10 @@
 dependencies:
 - name: celestia-node
   repository: file://../celestia-node
-  version: 0.4.0
+  version: 0.4.1
```

Comment:
if we rollback the dependency version will need to run here and redo the dependency update most likely

Reply:
Fixed in [60a150b](https://github.com/astriaorg/astria/pull/2085/commits/60a150b91b5d990cf9d0e9a6b85b8a788c40cbac).